### ISSUE TITLE
Outfit and ship warning when using German translation

### ISSUE THREAD

**Issue by Wuzzy2** [naev/naev#1085]
When I start Naev in German, a list of warnings appears in console. No warnings appear when I play in English.

There warnings appear:

```
Warning: [outfit_parseSBolt] Outfit 'BioPlasma Organ Stage 1' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'BioPlasma Organ Stage 2' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'BioPlasma Organ Stage 3' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'BioPlasma Organ Stage X' missing/invalid 'delay' element
Warning: [outfit_parseSLauncher] Outfit 'Electron Burst Cannon' missing 'delay' element
Warning: [outfit_parseSLauncher] Outfit 'TeraCom Mace Launcher' missing 'delay' element
Warning: [outfit_parseSLauncher] Outfit 'Unicorp Mace Launcher' missing 'delay' element
Warning: [outfit_parseSAmmo] Outfit 'Banshee Rocket' missing/invalid 'mass' element
Warning: [outfit_parseSAmmo] Outfit 'Convulsion Missile' missing/invalid 'mass' element
Warning: [outfit_parseSAmmo] Outfit 'Electron Energy Cell' missing/invalid 'mass' element
Warning: [outfit_parseSAmmo] Outfit 'EMP Grenade' missing/invalid 'mass' element
Warning: [outfit_parseSAmmo] Outfit 'Thurion Energy Cell I' missing/invalid 'mass' element
Warning: [outfit_parseSAmmo] Outfit 'Thurion Energy Cell II' missing/invalid 'mass' element
Warning: [outfit_parseSAmmo] Outfit 'Thurion Energy Cell III' missing/invalid 'mass' element
Warning: [outfit_parseSAmmo] Outfit 'Fury Missile' missing/invalid 'mass' element
Warning: [outfit_parseSAmmo] Outfit 'Headhunter Missile' missing/invalid 'mass' element
Warning: [outfit_parseSAmmo] Outfit 'Mace Rocket' missing/invalid 'mass' element
Warning: [outfit_parseSAmmo] Outfit 'Vengeance Missile' missing/invalid 'mass' element
Warning: [outfit_parseSBolt] Outfit 'Cheater's Laser Cannon' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Gauss Gun' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Heavy Ion Turret' missing/invalid 'track' element
Warning: [outfit_parseSBolt] Outfit 'Heavy Laser' missing/invalid 'track' element
Warning: [outfit_parseSBolt] Outfit 'Heavy Razor Turret' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Laser Cannon MK0' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Laser Cannon MK0' missing/invalid 'damage' element
Warning: [outfit_parseSBolt] Outfit 'Laser Cannon MK1' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Laser Cannon MK2' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Laser Cannon MK3' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Laser PD MK1' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Laser PD MK2' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Laser PD MK3' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Laser Turret MK1' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Laser Turret MK2' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Laser Turret MK3' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Neutron Disruptor' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Plasma Blaster MK1' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Plasma Blaster MK2' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Plasma Blaster MK3' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Plasma Cluster Cannon MK1' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Plasma Turret MK1' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Plasma Turret MK2' missing/invalid 'delay' element
Warning: [outfit_parseSBeam] Outfit 'Pulse Beam' missing/invalid 'delay' element
Warning: [outfit_parseSBeam] Outfit 'Pulse Beam' missing/invalid 'duration' element
Warning: [outfit_parseSBolt] Outfit 'Railgun' missing/invalid 'track' element
Warning: [outfit_parseSBolt] Outfit 'Railgun Turret' missing/invalid 'track' element
Warning: [outfit_parseSBolt] Outfit 'Razor MK1' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Razor MK2' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Razor MK3' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Razor Turret MK1' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Razor Turret MK2' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Razor Turret MK3' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Repeating Railgun' missing/invalid 'track' element
Warning: [outfit_parseSBolt] Outfit 'Shredder' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Turbolaser' missing/invalid 'track' element
Warning: [outfit_parseSBolt] Outfit 'Turreted Gauss Gun' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Turreted Vulcan Gun' missing/invalid 'delay' element
Warning: [outfit_parseSBolt] Outfit 'Vulcan Gun' missing/invalid 'delay' element
Warning: [outfit_load] Outfit 'Energy Dart' missing/invalid 'lockon' element
Warning: [outfit_load] Outfit 'Energy Missile' missing/invalid 'lockon' element
Warning: [outfit_load] Outfit 'Energy Torpedo' missing/invalid 'lockon' element
Warning: [outfit_load] Outfit 'TeraCom Imperator Launcher' missing/invalid 'ew_target' element
Warning: [outfit_load] Outfit 'Unicorp Caesar IV Launcher' missing/invalid 'ew_target' element
Warning: [outfit_load] Outfit 'Za'lek Reaper Launcher' missing/invalid 'ew_target' element
432 outfits loaded
Warning: [ship_parse] Ship 'Drone' missing 'time_mod' element
Warning: [ship_parse] Ship 'Hyena' missing 'time_mod' element
Warning: [ship_parse] Ship 'Proteron Derivative' missing 'time_mod' element
```

(Note: I have reset the translation for those warnings so you can read them because they were written in German, lol. Ummmm … maybe it was not a good idea to make the entire console debug output translatable … Just saying.)

I already posted the warnings somewhere else, but the context was something different (a crash). The crash was fixed since, but the warnings remain.

Again: These warnings ONLY appear if you play in German.

**Comment by ids1024** [comment]
The apparent cause of this seems to have been identified in the discussion on #1087 and 41c8f97e8ba405e5ffb79b5266a547cf40b56537 should fix it.

 @Wuzzy2 Can you confirm that fixes it, and you're able to run the game in German now?

**Comment by bobbens** [comment]
Should be fixed with #1087 .
